raspberry-mint-dev/src/www/index.html

```diff
@@ -0,0 +1,466 @@
+                placerat et at quam. Morbi fringilla condimentum magna, in ornare ligula aliquam quis.</p>
+        </section>
+
+        <section id="dev-link">
+            <h4>Link</h4>
+            <p>A <a href="#">link</a> to someplace else.</p>
+            <p><a href="#">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce vulputate magna sem, eget
+                rutrum eros consequat id. Aenean eu mauris condimentum, fermentum leo at, mollis diam. Pellentesque
+                hendrerit ornare sagittis. Cras imperdiet, nisi ac pulvinar ornare, elit magna semper risus, et posuere
+                mi justo ut sem. Morbi tincidunt odio non risus auctor facilisis in ut dolor. Vestibulum sed sapien
+                interdum, maximus leo quis, lacinia mi. Mauris ipsum orci, commodo quis lorem in, laoreet eleifend ante.
+                Etiam euismod neque sit amet diam pulvinar tincidunt vel vel urna.</a></p>
+        </section>
+
+        <section id="dev-scripts">
+            <h4>Scripts</h4>
+
+            <div class="row g-3">
+                <div class="col col-12 col-md-6 col-lg-4">
+                    <div class="card">
+                        <div class="card-body">
+                            <h5 class="card-title">Arabic</h5>
+                            <p class="card-text" lang="ar" style="direction: rtl;">مرحباً بكم في <span
+                                    lang="en">Betty</span>!</p>
+                        </div>
+                    </div>
+                </div>
+                <div class="col col-12 col-md-6 col-lg-4">
+                    <div class="card">
+                        <div class="card-body">
+                            <h5 class="card-title">Chinese, traditional (Cantonese)</h5>
+                            <p class="card-text" lang="yue">歡迎嚟到 <span lang="en">Betty</span>!</p>
+                        </div>
+                    </div>
+                </div>
+                <div class="col col-12 col-md-6 col-lg-4">
+                    <div class="card">
+                        <div class="card-body">
+                            <h5 class="card-title">Hebrew</h5>
+                            <p class="card-text" lang="he" style="direction: rtl;">ברוכים הבאים ל<span
+                                    lang="en">Betty</span>!
+                            </p>
+                        </div>
+                    </div>
+                </div>
+                <div class="col col-12 col-md-6 col-lg-4">
+                    <div class="card">
+                        <div class="card-body">
+                            <h5 class="card-title">Cyrillic (Ukrainian)</h5>
+                            <p class="card-text" lang="uk">Ласкаво просимо до <span lang="en">Betty</span>!</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+    </section>
+
+    <section class="mt-5">
+        <h2>Figures</h2>
```

Comment:
Remove this. We'll only end up using figures for the media gallery and those will not be reusable.